Specs/DateTime/Japanese/DatePeriodParser.json

```diff
@@ -0,0 +1,325 @@
+[
```

Comment:
Would you be able to help translating the other json specs? We recommend starting from the most basic types.
Other than that, the ones ending in Model are the most important ones.
Any case that still doesn't work can remain marked as notsupported. This is very valuable to track progress of proper support in a given language.